pkg/assets/assets.go

```diff
@@ -27,6 +25,26 @@ type Asset struct {
 // Assets is a list of assets.
 type Assets []Asset
 
+// NewRecursiveAssets walks through a directory and render each file as asset.
+func NewRecursiveAssets(dir string, data interface{}) (Assets, error) {
+	files, err := LoadFilesRecursively(dir)
+	if err != nil {
+		return nil, err
+	}
+
+	var as Assets
+	for path, bs := range files {
+		a, err := assetFromTemplate(path, bs, data)
+		if err != nil {
+			return nil, fmt.Errorf("failed to render %q: %v", path, err)
```

Comment:
i would probably just store the error in array and return errors at once... it will suck when I fix one error, rerun and get other error, etc... would be better to get all errors at once ;-)

Reply:
good idea

Reply:
done